### ISSUE TITLE
consider more fields in Search results? - was #394

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#802]
Eric D: A suggestion from a Coordinator now that we have a wider screen
Kirk: Please clarify if you are talking about the Search Results or the Database Contents. The title says one but the content seems to imply the other.
Nor do I feel we should be responding to single coordinator observations. None of these matters were raised when we asked for a systematic review by the 12 members of the test group,
Pat: Added to task list for intern: create mock-ups of potential changes to search results display (PC), and conduct formative user evaluation
Pat: Added field for testing/consideration: link to repository

**Comment by PatReynolds** [comment]
I have left this in 1.4 as the research may be completed as part of other user research in this phase.  Development is for 1.7

**Comment by PatReynolds** [comment]
Should include evaluation of #437 - Improvements to search results (currently 1.4)

**Comment by PatReynolds** [comment]
Should include evaluation of #570- Change search result display of witness to embedded collection

**Comment by PatReynolds** [comment]
Comments and questions from #394, moved here for clarity
Eric: The Register type is not needed in the first list of results
Display Parents first names in the first list as this is key to identifying a record
Display age in Baptism and Burial, again key to identifying the correct record
Pat: Added to task list for intern: create mock-ups of potential changes to search results display (PC), and conduct formative user evaluation
Pat: Added field for testing/consideration: link to repository
Kirk: "The search results provide a minimal sets of fields. All fields are shown in the detail.
Question is the minimal set too small? What should be added IF ANY. All is NOT a viable answer."

**Comment by edickens** [comment]
I don't disagree with that.
    
    Don't put too much on the first page as it was designed to be looked
    at on a Smartphone or Tablet.
    
    E
    
    On 21/06/2016 14:38, PatReynolds wrote:
    
    
      Comments and questions from #394,
        moved here for clarity
        Eric: The Register type is not needed in the first list of
        results
        Display Parents first names in the first list as this is key to
        identifying a record
        Display age in Baptism and Burial, again key to identifying the
        correct record
        Pat: Added to task list for intern: create mock-ups of potential
        changes to search results display (PC), and conduct formative
        user evaluation
        Pat: Added field for testing/consideration: link to repository
        Kirk: "The search results provide a minimal sets of fields. All
        fields are shown in the detail.
        Question is the minimal set too small? What should be added IF
        ANY. All is NOT a viable answer."
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.

**Comment by Captainkirkdawson** [comment]
We use different a display format for small screen devices such as smartphones.

**Comment by Captainkirkdawson** [comment]
Is this still relevant?

**Comment by PatReynolds** [comment]
Yes, I think so.  I saw the issue myself recently. I will add more info when I meet the problem again.

**Comment by Captainkirkdawson** [comment]
Remain deeply puzzled. Title implies adding more fields. But for what purpose?
@PatReynolds your response implies there is a problem/issue driving the story.
Remains ill defined
